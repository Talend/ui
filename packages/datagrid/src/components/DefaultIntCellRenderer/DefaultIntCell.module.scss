.td-cell-int {
	flex: 1;
	text-align: right;
}
