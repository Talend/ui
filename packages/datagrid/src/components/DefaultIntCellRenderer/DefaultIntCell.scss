.td-cell-int {
	text-align: right;
}
