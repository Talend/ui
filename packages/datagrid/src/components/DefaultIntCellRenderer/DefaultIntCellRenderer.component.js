import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

import theme from './DefaultIntCell.scss';

export default function DefaultIntCellRenderer({ data }) {
	return <div className={classNames(theme['td-cell-int'], 'td-cell-int')}>{data.value}</div>;
}

DefaultIntCellRenderer.propTypes = {
	data: PropTypes.shape({
		value: PropTypes.string,
	}),
};
