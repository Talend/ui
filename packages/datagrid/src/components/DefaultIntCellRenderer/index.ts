import DefaultIntCellRenderer from './DefaultIntCellRenderer.component';

export default DefaultIntCellRenderer;
