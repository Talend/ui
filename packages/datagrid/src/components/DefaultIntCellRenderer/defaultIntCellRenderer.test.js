import React from 'react';
import { shallow } from 'enzyme';

import DefaultIntCellRenderer from './DefaultIntCellRenderer.component';

describe('#DefaultDateCellRenderer', () => {
	it('should render DefaultDateCellRenderer', () => {
		const wrapper = shallow(<DefaultIntCellRenderer data={{ value: 42.42 }} />);

		expect(wrapper.getElement()).toMatchSnapshot();
	});
});
