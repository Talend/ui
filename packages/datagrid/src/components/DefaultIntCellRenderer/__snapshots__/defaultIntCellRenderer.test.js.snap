// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#DefaultDateCellRenderer should render DefaultDateCellRenderer 1`] = `
<div
  className="theme-td-cell-int td-cell-int"
>
  42.42
</div>
`;
